client/src/styles/OrderField.scss

```diff
@@ -0,0 +1,13 @@
+.ckan-resource__order.field .form__field-holder {
+  input {
+      max-width: 6em;
```

Comment:
Indentation should be 2 spaces not 4 (is our linter broken?)

Reply:
.editorconfig seems broken in my IDE!
I'll tidy up in next PR :)